frontend/src/pages/auth/components/auth-layout/auth-layout.tsx

```diff
@@ -0,0 +1,40 @@
+import { type ReactNode } from 'react';
+
+import {
+  AUTH_FORM_WRAPPER_FLOATING_LETTERS,
+  HERO_FLOATING_LETTERS,
+} from './libs/constants.js';
+import { type FloatingLetter } from './libs/types/floating-letter.type.js';
+import styles from './styles.module.scss';
+
+type Properties = {
+  children: ReactNode;
+};
+
+const AuthLayout: React.FC<Properties> = ({ children }) => {
+  const renderFloatingLetters = (letters: FloatingLetter[]): ReactNode[] =>
+    letters.map((item) => (
+      <div
+        key={item.key}
+        style={{ ...item.position }}
```

Comment:
redundant destructurization as you pass nothing else there
```suggestion
        style={item.position}
```